lib/diagnostic.js

```diff
@@ -28,7 +37,10 @@
 // Attributes: name, address, numberOfPets
 
 // Replace null with your answer
-let owner = null;
```

Comment:
We're looking for something with the following form.
```javascript
let owner = {
  name: 'Someone',
  address: 'Some place',
  numberOfPets: 1
};
```
What you gave was a nested object with ```owner.Attributes``` as the object that we're looking for.